docs/components/buttons/SidebarCategorizationButton.js

```diff
@@ -10,23 +10,35 @@ type Props = {|
 |};
 
 export default function SidebarCategorizationButton({ onClick, sidebarOrganisedBy }: Props): Node {
-  const sidebarOrganisedByCopy =
-    sidebarOrganisedBy === 'categorized' ? 'Alphabetical' : 'Categorize';
+  const [selected, setSelected] = React.useState(sidebarOrganisedBy);
```

Comment:
Why is this local state necessary? It seems like it's duplicating what's being handled in the parent, and could possibly lead to a buggy UI from drift